cloud/tests/conftest.py

```diff
@@ -4,6 +4,11 @@
 
 
 class CloudTestHelper:
+    
+    cpu_training_image = '369469875935.dkr.ecr.us-east-1.amazonaws.com/autogluon-nightly-training:cpu-latest'
```

Comment:
sanity check: are the tests running in `us-east-1` region?
(making sure none of the parts are making cross-region calls).

Reply:
Yes they are